core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.css

```diff
@@ -407,7 +407,7 @@
   color: #009688;
 }
 .oppia-top-navigation-bar .oppia-navbar-dropdown .nav-link:hover {
-  color: #888;
+  background-color: #d7d7d7;
```

Comment:
?

Reply:
@heyimShivam as you can see the before and after video of this pull request https://github.com/oppia/oppia/pull/17490#issue-1599841419
here on clicking the enter on the profile icon of the navbar the dropdown is opening, but we are not able to see what is selected clearly as there is no clear color indication, so by this I have changed the background so that it is visible what is selected.